modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarCollapse.es.js

```diff
@@ -110,31 +110,30 @@ class CriteriaSidebarCollapse extends Component {
 									<li className="empty-message">
 										{Liferay.Language.get('no-results-were-found')}
 									</li> :
-									active && filteredProperties.length &&
```

Comment:
What happened to these checks?

Also, its gonna be easier to read this code if you separate this into two chunks. See code below

```js
{active && filteredProperties.length === 0 &&
	<li className="empty-message">
		{Liferay.Language.get('no-results-were-found')}
        </li>
}

{active && filteredProperties.length &&
	filteredProperties.map(
		({label, name, options, type}) => {
			const defaultValue = getDefaultValue(
				{
					label,
					name,
					options,
					type
				}
			);

			return (
				<CriteriaSidebarItem
					className={`color--${key}`}
					defaultValue={defaultValue}
					key={name}
					label={label}
					name={name}
					propertyKey={key}
					type={type}
				/>
			);
		}
	)
}
```

Reply:
Ok, I'll send it in a couple of minutes :) Sorry for sent it to Brian, I reviewed with @andresfulla and he checked. We'll follow the correct workflow now on

Reply:
No worries! I am fine with changing the review process if it becomes a roadblock for you guys. Or if there is ever anything that is high priority, just ping me on slack and I will get to it asap.